HeterogeneousCore/AlpakaInterface/interface/atomicMaxPair.h

```diff
@@ -0,0 +1,38 @@
+#ifndef HeterogeneousCore_AlpakaCore_interface_atomicMaxPair_h
+#define HeterogeneousCore_AlpakaCore_interface_atomicMaxPair_h
+#include <alpaka/alpaka.hpp>
+
+#include "FWCore/Utilities/interface/bit_cast.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+// Note: Does not compile with ALPAKA_FN_ACC on ROCm
```

Comment:
Well, given the `#ifdef`, the function is defined both for the host and the device, so it makes sense that it doesn't compile as a pure `__device__` function.

The canonical alpaka approach would be to have different specialisation for each accelerator - but we can clean this up later.